plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -141,6 +141,23 @@ protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMapp
         return Optional.of(dataMappingTestSetup);
     }
 
+    @Override
+    public void testRenameColumnName(String columnName)
+    {
+        // TODO: Fix failure
+        if (columnName.equals("a.dot") || columnName.equals("a\"quote")) {
+            assertThatThrownBy(() -> super.testRenameColumnName(columnName))
+                    .hasMessageContaining("Syntax error parsing SQL identifier");
+            throw new SkipException("TODO");
+        }
+        if (columnName.equals("an'apostrophe")) {
+            assertThatThrownBy(() -> super.testRenameColumnName(columnName))
+                    .hasMessageContaining("Incorrect syntax near 'apostrophe'.");
```

Comment:
Errs mentioning "syntax" indicate like there is SQL injection happening. 
is there a problem here?